test/verify/check-machines

```diff
@@ -483,6 +483,78 @@ class TestMachines(MachineCase):
 
         b.wait_in_text("#vm-subVmTest1-network-2-state span", "up")
 
+    def testVCPU(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+        b.click("tbody tr th") # click on the row header
+
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+        # Shut off VM for applying changes after save 
```

Comment:
There have not been any actual changes. As before, this actually should test changes while the VM is running, make sure that the "will be applied after reboot" message appears, and the changes are taken into account after reboot. This can happen at the end of the current test when the VM is rebooted and running again.

Reply:
Testing for message "will be applied after reboot", already there, on line 547.